state/state-network.md

```diff
@@ -209,7 +209,7 @@ content_for_retrieval := Container(node: TrieNode)
 These data types represent a node from an individual contract storage trie.
 
 ```
-storage_trie_node_key  := Container(address: Address, path: Nibbles, node_hash: Bytes32)
+storage_trie_node_key  := Container(address_hash: keccak(Address), path: Nibbles, node_hash: Bytes32)
```

Comment:
nit: In my opinion, this should be: `address_hash: AddressHash` and in the *"Helper Data Types"* section above, you would define `AddressHash := Bytes32` with brief explanation.